### DISCUSSION TITLE
Galactic Habitability Approximation (Suggested Guideline)

### DISCUSSION THREAD

**Discussion by MasterOfGrey** [endless-sky/endless-sky#8460] [Ideas]
### Principle
Before I get into detail: Yes, gameplay trumps realism. Yes, narrative trumps realism.

_However,_ I'm personally in the camp of believing that at least loosely conforming to realism can provide added weight to gameplay and narrative considerations. As such this is put forward as a _suggestion_ for consideration when placing alien species or considering expansions to the map.

![galaxy_habitability_nobound_1200](https://user-images.githubusercontent.com/60949828/112707871-7161bc00-8efe-11eb-95cf-af7aa89b8c2d.png)

I have colour shaded the galaxy roughly according to how the combination of ambient heat, stellar radiation, and celestial volatility would _theoretically_ influence the development of sapient life. This is roughly how the galaxy splits up:
- Dark Red: This is mega-death zone nothing habitable should be here, even barren planets is a stretch.
- Bright Red: This is regular death zone. Nothing habitable should be here but planets should be somewhat present. High risk of neighbouring supernova.
- Orange: This is hardcore unpleasant. Life is possible here, with a bit of luck and a minor suspension of disbelief, but it would be strange, different, and designed to tolerate harsh and strange climates. The volatility is such that evolution would happen quickly, possibly sneaking in complex life before a supernova wipes it out. (Only extremophiles on earth might be comfortable here though so life would be _quite_ different.)
- Yellow: This is generally a bad time. Not volatile enough to rely on it, not gentle enough by far to allow more normal types of life to safely develop. This is why I would put the exiles here in a re-work of the core - a world like Far en Lai _could_ exist here and support life like we know it, but it would be unlikely, and it would be the sheltered gem in a space otherwise unpleasant to life and technology developed further away from the core.
- Light Green (first full ring around the core): By this point you've got somewhat normal galactic conditions. However, this means the usual millions of years to evolve complex life. It's close enough to the core that a stray gamma ray burst is likely to happen sometime in that process and kill off evolution. Density of systems high, habitable planets possible, actual sentience arising unlikely. (Great for aliens to move into, but not necessarily arise within.)
- Green: This is the densest useful part of the galaxy, and the part most favourable to life arising, and just exciting enough to drive development quickly (uncompetitive species would likely be wiped out be a disaster before reaching the stars). Note humanity is very close by, and Korath are almost entirely within this band.
- Blue: The neighbourhood of the galaxy, gentle, stable, pleasant, has a few bright spots of excitement. Generally the best place for life to be comfortable, though spawn chance is lower than green, but survival odds are vastly higher.
- Purple, the boondocks of the galaxy, quiet, isolated, conditions are pretty much determined by the status of the local star with very few external inputs.

### Conclusion
It is worth noting that, with the exception of coreward Syndicate space, everything else in the game _already_ conforms to this.

It is by no means necessary that people conform to this; but I hope it provides people considering things that are as yet undecided some insight and context for different regions of the galaxy.

**Comment by MCOfficer** [comment]
Since this is an argument for realism, can you cite some sources? I've never heard of such a categorization on a galactic scale.

**Comment by Vedvod** [comment]
I can not speak for where MasterOfGrey has obtained their information, but I can provide several citations ([https://arxiv.org/abs/astro-ph/0103165](https://arxiv.org/abs/astro-ph/0103165), John Billingham's _Life in the universe_, and [https://arxiv.org/abs/astro-ph/0401024](https://arxiv.org/abs/astro-ph/0401024), amongst others) for **_projections/spectulations_** of how life in the galaxy would be distributed/develop. Also worth reading is [Wikipedia's Galactic Habitable Zone article](https://en.m.wikipedia.org/wiki/Galactic_habitable_zone). Once again, these are speculation, though they are definitely well-based speculations. 
It would be interesting to somehow have this concept of galactic habitable zones be somehow included in the endless sky galaxy, but I can't really see any real implementation for it, since, as mentioned, everything already makes sense. That's just my (bad) opinion though, ignore it. Hope this maybe sorta slightly helped (maybe?).

**Comment by Zireael07** [comment]
https://en.wikipedia.org/wiki/Galactic_habitable_zone

The Wikipedia page has a mile-long list of scientific articles at the bottom.

EDIT: Ninja'd by @Vedvod :P

**Comment by MasterOfGrey** [comment]
It looks like Vedvod and Zirael have capably linked to a collection of relevant sources. What you see here MCO is simply a loose reconciliation of the game's galaxy and existing state, with the collection of general knowledge I've picked up from reading some of those sources in the past.

> but I can't really see any real implementation for it, since, as mentioned, everything already makes sense

Everything *_except_* the existing core makes sense, which I have plans for. This is less about the existing stuff though, and more about promoting some consistency in how things are laid out with the addition of new alien races in the future.

**Comment by MasterOfGrey** [comment]
Btw @Zireael07 if no one has invited you before, we'd love you to join the discord server were most content creation discussions occur. Link: https://discord.gg/ZeuASSx
